### ISSUE TITLE
Lint fails with "same as" test

### ISSUE THREAD

**Issue by neilime** [asm89/twig-lint#33]
Hi,

here is an unexpected lint error :

```twig
{% if foo.attribute is same as(false) %}
    the foo attribute really is the 'false' PHP value
{% endif %}
```

Error : 

```bash
>> 1      {% if foo.attribute is same as(false) %}
           >> Unexpected token "name" of value "as" ("end of statement block" expected).
              2          the foo attribute really is the 'false' PHP value
              3      {% endif %}
```

However it works with the test "sameas".

(Twig doc : http://twig.sensiolabs.org/doc/2.x/tests/sameas.html)

Thanks for your help.

**Comment by seanhamlin** [comment]
Experiencing the same issue. This is basically a show stopper for me to use this linter.

```
>> 10             {% if count > 0 and not type is same as('total') %}
>> Unexpected token "name" of value "as" ("end of statement block" expected)
```

**Comment by asm89** [comment]
Is this still an issue on the most recent version?

**Comment by bronzehedwick** [comment]
@asm89 Yes I can confirm this is still an issue on version `1.0.3`.
